src/Microsoft.TemplateEngine.Edge/Settings/TemplateMatchInfo.cs

```diff
@@ -58,4 +62,39 @@ public void AddDisposition(MatchInfo newDisposition)
 
         public bool IsPartialMatch => MatchDisposition.Any(x => x.Kind != MatchKind.Mismatch);
     }
+
+    internal class TemplateMatchInfo2 : Abstractions.TemplateFiltering.ITemplateMatchInfo
```

Comment:
I'm confused what are we trying to do here? We made 90% of things internal, we made major changes in Edge like how Installing SDK templates work and all that, why are we so careful about this class?

Reply:
I haven't removed anything from Edge when working on public APIs, marked obsolete instead. 
Let's discuss whether we drop everything or make obsolete.